.github/workflows/build-and-push.yml

```
@@ -0,0 +1,23 @@
+name: "docker"
+on:
+  push:
+    branches:
+      - master
+  pull_request: 
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout source code at current commit"
+        uses: actions/checkout@v1
```

```suggestion
        uses: actions/checkout@v2
```

why not use the latest version